web/src/components/Footer/index.tsx

```diff
@@ -20,7 +20,7 @@ import { DefaultFooter } from '@ant-design/pro-layout';
 
 export default () => (
   <DefaultFooter
-    copyright="2021 Apache APISIX"
+    copyright=`${new Date().getFullYear()} Apache APISIX`
```

Comment:
```suggestion
    copyright={`${new Date().getFullYear()} Apache APISIX`}
```

Ahh, missed suggesting a wrap. CI should be fine now.